app/src/main/java/space/taran/arkretouch/presentation/edit/MoreOptionsPopup.kt

```diff
@@ -0,0 +1,104 @@
+package space.taran.arkretouch.presentation.edit
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Popup
+import androidx.compose.ui.window.PopupProperties
+import space.taran.arkretouch.R
+import space.taran.arkretouch.presentation.picker.toPx
+
+@Composable
+fun MoreOptionsPopup(
+    onShareClick: () -> Unit,
+    onSaveClick: () -> Unit,
+    onDismissClick: () -> Unit
+) {
+    Popup(
+        alignment = Alignment.TopEnd,
+        offset = IntOffset(
+            -8.dp.toPx().toInt(),
+            8.dp.toPx().toInt()
+        ),
+        properties = PopupProperties(
+            focusable = true
+        ),
+        onDismissRequest = onDismissClick,
+    ) {
+        Column(
+            Modifier
+                .background(
+                    Color.LightGray,
+                    RoundedCornerShape(8)
+                )
+                .padding(8.dp)
+        ) {
+            Row {
```

Comment:
Looks like column is redundant.

Reply:
It's deliberate, may we will add some Rows in future when more buttons are added